styles/mixins/menu-slide.css

```diff
@@ -102,4 +102,38 @@ title: Mobile Menu
       }
     }
   }
+  &.mobile-only {
+    & .menu-btn {
+      & + label {
+        display: none;
+
+        @media (--tablet) {
+          display: block;
+        }
+      }
+      &:checked {
+        & ~ .menu-container {
+          right: 60%;
+          overflow-y: auto;
+        }
+      }
+      @media (--tablet) {
+        & ~ .menu-container {
+          position: fixed;
```

Comment:
We'd need to use something like [davidtheclark/postcss-simple-extend](https://github.com/davidtheclark/postcss-simple-extend) to fix that.  Not sure we want to add this for just this one use case.